lib/stacker/stack.rb

```diff
@@ -185,6 +185,23 @@ def describe_change_set
           sleep ((6 - retries) * 3)
         end
       end
+      resp_status = ""
+      while resp_status != "CREATE_COMPLETE"
```

Comment:
i think this loop achieves what the loop above is meant to, so we should get rid of the one above. the one bit of functionality we're missing here is grabbing the changes list once creation is complete

Reply:
Ack, addressed and updated